components/AlertDialog.tsx

```diff
@@ -0,0 +1,88 @@
+import { useCallback, useState } from "react"
+import { Modal, View, StyleSheet, Platform } from "react-native"
+import CustomText from "@/components/ui/CustomText"
+import { theme } from "@/theme"
+import { CustomButton } from "@/components/ui/Button"
+
+interface AlertDialogProps {
+  title: string
+  description: string
+  cancelLabelText?: string
```

Comment:
This name is limiting, since this won't be used to cancel only.
```suggestion
  labelText?: string
```